### DISCUSSION TITLE
Crossplay v2.1.1 for Terraria 1.4.4.8.1

### DISCUSSION THREAD

**Discussion by Moneylover3246** [Moneylover3246/Crossplay#49] [Announcements]
## Overview
- Fixes the functionality of the plugin (@SirMonkeyBoy) 

<hr /><em>This discussion was created from the release <a href='https://github.com/Moneylover3246/Crossplay/releases/tag/2.1.1'>Crossplay v2.1.1 for Terraria 1.4.4.8.1</a>.</em>

**Comment by AleJHG** [comment]
Hello, thank for de update the crossplay launcher, but terraria has a new versión 1.4.4.9, could you update the version of the crossplay file? pls
